documentation/docs/11-ssr-and-javascript.md

```diff
@@ -6,21 +6,7 @@ By default, SvelteKit will render any component first on the server and send it
 
 You can control each of these on a per-app or per-page basis. Note that each of the per-page settings use [`context="module"`](https://svelte.dev/docs#script_context_module), and only apply to page components, _not_ [layout](#layouts) components.
 
-If both are specified, per-page settings override per-app settings in case of conflicts. Each setting can be controlled independently, but `ssr` and `hydrate` cannot both be `false` since that would result in nothing being rendered at all.
```

Comment:
It might be worth keeping a note that `ssr` and `hydrate` cannot both be `false` since that would result in nothing being rendered at all. I've seen people trip up on that a number of times. Could move it under `hydrate` perhaps

Reply:
agree, done